tls/s2n_record_write.c

```diff
@@ -110,6 +110,11 @@ int s2n_record_write_protocol_version(struct s2n_connection *conn)
         record_protocol_version = MIN(record_protocol_version, S2N_TLS10);
     }
 
+    /* In accordance to TLS 1.3 spec, https://tools.ietf.org/html/rfc8446#section-5.1
+     * tls record version should never be greater than 33 (legacy TLS 1.2 version).
```

Comment:
Q: Is there a reason behind using `33` and not `0x0303` as specified in the spec?

Reply:
ah, no difference, just wording - 33 is decimal and 0x0303 is hexadecimal.

Reply:
I see, thank-you.